src/context/derived_fields.rs

```diff
@@ -7,18 +7,17 @@ use crate::logging;
 
 /// This function checks whether all the necessary data is present in the store to avoid linking
 /// entities to other non existent entities which may cause serious collision problems later
-pub(crate) fn insert_derived_field_in_store<C: graph::blockchain::Blockchain>(
-    context: &mut MatchstickInstanceContext<C>,
+pub(crate) fn insert_derived_field_in_store(
+    store: &mut HashMap<String, HashMap<String, HashMap<String, Value>>>,
```

Comment:
Here you can replace `HashMap<String, HashMap<String, HashMap<String, Value>>>` with `super::Store` to reuse the `Store` you have defined in the parent module